lib/plugins/horse.js

```diff
@@ -0,0 +1,80 @@
+const { callbackify } = require('../promise_utils')
+
+module.exports = inject
+
+function inject (bot) {
+  const metadataMap = {
+    base: {
```

Comment:
is this correct for all versions ?

Reply:
Good call, looks like it is very likely not. Seems like another thing to be added in minecraft-data /._.